<launch>
  <node name="motion_control" pkg="motion_control" type="motion_control_node" >
    <remap from="scan" to="/ropod/laser/scan"/>
    <rosparam file="$(find motion_control)/config/control.yaml"/>
    <rosparam file="$(find motion_control)/config/costmap.yaml" ns="local_map" />
  </node>
</launch>

